crates/bevy_app/src/app_builder.rs

```diff
@@ -244,9 +244,11 @@ impl AppBuilder {
     pub fn add_startup_system_set(&mut self, system_set: SystemSet) -> &mut Self {
         self.add_startup_system_set_to_stage(StartupStage::Startup, system_set)
     }
-  
-    /// Adds a system to the [startup stage](Self::add_default_stages) identified by
-    /// `stage_label`.
+
+    /// Adds a system to the [startup schedule](Self::add_default_stages), in the stage
+    /// identified by `stage_label`.
+    ///
+    /// Stage label must be of type [`StartupStage`].
```

Comment:
The `must` is wrong here, as you can insert your own stages, that have a different type.
Perhaps instead 
```suggestion
    /// Stage label is typically of the type [`StartupStage`].
```

Reply:
Thanks, nice catch 😀.

I went a step further: I specified that it must refer to a stage inside the startup schedule. I also changed it in other occurences.